apps/web/src/components/Shared/Modal/ReportProfile/index.tsx

```diff
@@ -4,32 +4,32 @@ import { t } from '@lingui/macro';
 import { type FC, useState } from 'react';
 import { object, string } from 'zod';
 
-interface ReportProfileProps {
-  profile: Profile;
-}
-
 const reportProfileSchema = object({
   report: string().max(300, {
     message: t`Report should not exceed 300 characters`
   })
 });
 
-const Report: FC<ReportProfileProps> = ({ profile }) => {
+interface ReportProfileProps {
+  profile: Profile | null;
+}
+
+const ReportProfile: FC<ReportProfileProps> = ({ profile }) => {
   const [isRadioSelected, setIsRadioSelected] = useState(false);
 
   const form = useZodForm({
     schema: reportProfileSchema
   });
 
   return (
-    <div className="flex flex-col space-y-2 p-3">
+    <div className="flex flex-col space-y-2 p-5">
       <Form
         form={form}
         onSubmit={() => {
           alert('Submitted' + profile?.id);
```

Comment:
@spantheslayer Can you also add what radio user is selected? 🙇🏼 

make sure to use enum here

Reply:
when using padding of 5 , the allignment goes unexpected, thus I kept it at 3.